README.md

```diff
@@ -14,6 +13,72 @@ Contents
 - [Acknowledgments](#acknowledgments)
 - [License](#license)
 
+## Motivation
+
+[Manta Style](https://github.com/Cryrivers/manta-style/issues/1) generates "real" _(enough)_ mock data from [TypeScript](http://www.typescriptlang.org/) type definitions.
+
+With _Manta Style_, you can start implementing feature once your data schema is defined. 
+But _Manta Style_ is more than just that.
+
+### Generates mock data directly from your type declarations
+
+Right now Manta Style supports only TypeScript. We are working on FlowType.
+<!-- some more words goes here @TODO wgao19 -->
+
+### Mock data with respect to real world scenario such as past dates, addresses, names
+
+```ts
+interface User {
+  /**
+   * @faker {{internet.userName}}
+   * 
+   */
+  userName: string; // Amina.Langosh49
+
+  /**
+   * @timestamp past
```

Comment:
this syntax has been changed. i'll update it later.

Reply:
oh it's not in this repo so i cannot idea. new syntax is `@faker date.past`

Reply:
I think you can edit in your main repo, so before I merge I will resolve conflict and use yours.

Reply:
try rebasing.